packages/main/src/plugin/kubernetes/kubernetes-client.ts

```diff
@@ -270,6 +270,8 @@ export class KubernetesClient {
     const statesExperimental = this.experimentalConfigurationManager.isExperimentalConfigurationEnabled(
       'kubernetes.statesExperimental',
     );
+    this.telemetry.track('kubernetesExperimentalMode', { enabled: statesExperimental });
```

Comment:
question: is it easy in amplitude to get a ratio of the people having the experimental mode enabled with this?